channeldb/channel.go

```diff
@@ -3,6 +3,7 @@ package channeldb
 import (
 	"bytes"
 	"encoding/binary"
+	"errors"
```

Comment:
Lingering TODO in the commit.

Reply:
I think it makes sense to NOT do it in the migration (the only difference would be that we could remove the EOF check below), so will remove the TODO from the commit message if no protests.

Reply:
SGTM